Sources/XCHammer/XcodeTarget.swift

```diff
@@ -593,6 +595,31 @@ public class XcodeTarget: Hashable, Equatable {
                 break // These attrs are not related to XCConfigs
             case .binary:
                 break // Explicitly not handled since it is a implicit target we don't intend to handle
+            case .swift_language_version:
+                guard let swiftVersion = value as? String else {
+                    break
+                }
+                settings.swiftVersion <>=  First(swiftVersion)
+            case .has_swift_dependency, .has_swift_info:
+                guard let swiftVersion = try? ShellOut.shellOut(to: "xcrun swift package tools-version") else {
+                    break
+                }
+                settings.swiftVersion <>=  First(swiftVersion)
+            case .swiftc_opts:
+                if let coptsArray = value as? [String] {
+                    let processedOpts = coptsArray.map { opt -> String in
```

Comment:
Do we need to patch these for `bazel-genfiles` ( and for `external` in a followup ). We should have an example of a `swift` genfile in a `swift_library` in a sample. Consider either adding this to `UrlGet` or just creating a new project where we can add all the swift edge cases.

Reply:
Probably patch this for external with the PR that handles external repos. Adding a new example is a good idea

Reply:
lets do this in a follow up PR